handler/name.go

```diff
@@ -0,0 +1,16 @@
+package handler
+
+import (
+	"fmt"
+	"strings"
+)
+
+func Name(r Interface) string {
+	split := strings.Split(fmt.Sprintf("%#v", r), ".")
+
+	if len(split) < 2 {
+		return "n/a"
```

Comment:
IMO this should panic here. It has tests and with VOO we don't update libraries for past releases.

Reply:
Added panic.